06_neighborhoods/Transportation/transit_trips_county.R

```diff
@@ -0,0 +1,203 @@
+###################################################################
+
+# Transit Metrics: Transit Trips
+# Tina Chelidze 
+# 2023-2024 Mobility Metrics update
+# Process (for overall, not subgroup, data):
+# 1. For County: Import and reshape data
+# 2. QC checks
+# 3. Add Data Quality marker
+# 4. Export final files
+
+###################################################################
+
+
+# Libraries you'll need
+library(tidyr)
+library(dplyr)
+library(readr)
+library(ggplot2)
+library(tidyverse)
+library(purrr)
+
+# SUMMARY-LEVEL VALUES
+# Raw data pulled from https://htaindex.cnt.org/download/
+# the Housing and Transportation (H+T) Affordability Index from the Center for Neighborhood Technology (CNT)
+
+###################################################################
+
+# 1. Import all the files (and/or combine into one file) with only the relevant variables and years
+
+# FOR COUNTIES - import the raw data files
+# Unlike for Places and Tracts, the CNT website allows you to download raw data for all counties at once
+# so we just import the all-county file for the two available years; 2015 and 2019
+# 2015
+transport_county_2015 <- read_csv("C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/htaindex2015_data_counties.csv")
+
+# create correct FIPS columns
+transport_county_2015 <- transport_county_2015 %>%
+  mutate(
+    state = substr(county, start = 2, stop = 3),
+    county = substr(county, start = 4, stop = 6)
+  )
+
+# keep only variables of interest
+transit_trips_county_2015 <- transport_county_2015 %>%
+  select(state, county, blkgrps, population, households, transit_trips_80ami)
+
+
+#2019
+transport_county_2019 <- read_csv("C:/Users/USERNAME/Box/Lab/Projects/Gates Upward Mobility Framework/Outreach and Tools/Data/Metrics_2024_round/Transportation/htaindex2019_data_counties.csv")
+transport_county_2019 <- transport_county_2019 %>%
+  select(county, blkgrps, population, households, transit_trips_80ami)
+
+# create correct FIPS columns
+transport_county_2019 <- transport_county_2019 %>%
+  mutate(
+    state = substr(county, start = 2, stop = 3),
+    county = substr(county, start = 4, stop = 6)
+  )
+
+# keep only variables of interest
+transit_trips_county_2019 <- transport_county_2019 %>%
+  select(state, county, blkgrps, population, households, transit_trips_80ami)
+
+# Compare to our official county file to make sure we have all counties accounted for
+# Bring in the official county file
+counties <- read_csv("C:/Users/USERNAME/Documents/GitHub/mobility-from-poverty/geographic-crosswalks/data/county-populations.csv")
```

Comment:
Since we have a .rproj that we ask people to use, this should be soft coded instead of hard coded.

`counties <- read_csv("geographic-crosswalks/data/county-populations.csv")`